src/ngx_http_lua_misc.c

```diff
@@ -58,6 +62,40 @@ ngx_http_lua_ngx_req_is_internal(lua_State *L)
 
 
 static int
+ngx_http_lua_ngx_req_get_intercept_log(lua_State *L)
```

Comment:
I'm sorry, this is useless now. I wrote the comment long ago, and I didn't realize it just now.